RU/wer.md

```diff
@@ -0,0 +1,33 @@
+
+# WER results
+This file presents the average (avg.) WER scores of the three datasets tested within the [HoMed](https://homed.ruhosting.nl/) project (2021-2024).
+The concrete results (INS, DEL, SUB) of each one of the files of the three datasets are included in the .sys files generated with [SCLITE](https://github.com/usnistgov/SCTK).
+
+
+1. Medicijnjournaal (**MJ**): 35 files
```

Comment:
Could you format the results in a table format? See UT/N-Best/wer.md for an example

Additionally, I imagine there will be some gaps since some datasets haven't been tested with some models. In that case, you may leave the gap there or add a `-`